sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -27,7 +27,7 @@ internal static List<TelemetryItem> OtelToAzureMonitorTrace(Batch<Activity> batc
             foreach (var activity in batchActivity)
             {
                 var monitorTags = EnumerateActivityTags(activity);
-                telemetryItem = new TelemetryItem(activity, ref monitorTags, roleName, roleInstance, instrumentationKey);
+                telemetryItem = new TelemetryItem(TelemetryItem.s_telemetryItem_Name_Mapping[activity.GetTelemetryType()], activity, ref monitorTags, roleName, roleInstance, instrumentationKey);
```

Comment:
Should we move `s_telemetryItem_Name_Mapping` to this file and rename it to a shorter name?